tests/snippets/stdlib_os.py

```diff
@@ -143,6 +143,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 	print(stat_res.st_gid)
 	print(stat_res.st_size)
 	assert stat_res.st_size == len(CONTENT2) + len(CONTENT3)
+	print(stat_res.st_atime)
+	print(stat_res.st_ctime)
+	print(stat_res.st_mtime)
+	# test that it all of these times are greater than the 10 May 2019, when this test was written
+	assert stat_res.st_atime > 1557500000
```

Comment:
I think you should check here that `test1` file is older then `test2` file.
I would also check that after changing the file the modification time is increased.